src/modules/getthis/components/GetThisOptionList/index.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+import config from '../../../../config'
+import { DetailsList } from '../../../core'
+import GetThisOption from '../GetThisOption'
+
+class GetThisOptions extends React.Component {
+  render() {
+    const { record } = this.props
+
+    if (record && record.getthis) {
+      const { status, options } = record.getthis
+
+      if (status === 'Success') {
+
+        if (options.length === 0) {
+          return (
+            <section className="card get-this-section">
+              <h2 className="get-this-section-heading">How would you like to get this item?</h2>
```

Comment:
Since we already know that there aren't any options, we could possibly rephrase this.

Reply:
Yeah. I kept the heading the same since in the designs it seemed static. Although it does seem misleading since we know there are no options. I am hoping the heading anchors the section and then the content below describes the state, such as options, errors, etc.